content/vendor/guides/ha-guide.md

```diff
@@ -0,0 +1,411 @@
+
+![PostgresDesribe](/images/guides/kots/ha-cluster-postgres-describe.png)
+
+The first thing to check is to ensure you have followed the [configuring ekco] section. Ekco helps with ensuring that any resources that were running on the stopped node are properly deleted and moved to a different node. Without ekco, the claim on the PVC associated to the Postgres pod that was running on the stopped node sometimes doesn't get properly cleared and when the new pod gets scheduled on another node, it is not able to lay a claim to the volume.
+
+### Other Persistent Data Issues
+
+The default configuration of the Kurl installer includes the [rook add-on](https://kurl.sh/docs/add-ons/rook), which creates and manages a [Ceph cluster](https://docs.ceph.com/docs/master/rados/) along with a storage class for provisioning Persistent Volume Claims (PVCs). If you are running into other issues with persisitent storage, following are some helpul commands to check the health of ceph in your cluster. Keep in mind that the screenshots below show a `healthy` cluster that has all nodes up and running. 
+
+All related pods are runnning in the rook-ceph namespace, and a good place to start to troubleshoot any issues. By running `kubectl get pods -n rook-ceph` you should get an output similar to the one below:
+
+![PostgresDesribe](/images/guides/kots/ha-cluster-rook-ceph-pods.png)
+
+One thing to note here is that you will see pods running on all three nodes. Orchestrating all of the data accross all nodes is the `rook-ceph-operator-...` pod. To troubleshoot issues, we'll need to exec into this pod in order to run some commands. For example, to exec into the pod on the screenshot above, I would run `kubectl exec -it rook-ceph-operator-6fbfdccc57-p477z`. As you exec into the pod, you may see a few errors but you can igonre them.
+
+The check the status of ceph, run `ceph status`:
+
+![CephStatus](/images/guides/kots/ha-cluster-ceph-status.png)
+
+The output provides you with a high level overview of the ceph cluster. Note the value for 'health' is set to `HEALTH_OK`. When a node is unresponsive, the 'health' is set to `HEALTH_WARN`.
+
+To check the [Object Storage Deamons (OSDs)](https://docs.ceph.com/docs/master/man/8/ceph-osd/) in more detail, run `ceph osd status`
+
+![OSDStatus](/images/guides/kots/ha-cluster-osd-status.png)
+
+To get another look at the OSDs, run `ceph osd tree`
+
+![OSDTree](/images/guides/kots/ha-cluster-osd-status.png)
+
+## Next Steps
+
+The guide covers a basic 3 node HA cluster. However, there are other layers of complexity that could be added depending on your use case:
+
+- Add worker nodes to the cluster and run same tests.
+- Set up https access 
+- Setting up Postgres in HA mode and compare down time against a single Postgres deployment.
```

Comment:
Should this be `Set` to match the phrasing of the first two bullets?

Reply:
yes, done